src/frontend/js/components/DashboardCourseRun/index.tsx

```diff
@@ -0,0 +1,30 @@
+import { CourseRun } from '../../types';
```

Comment:
You delete this component in the next commit. You could squash these two modifications to only show the final version of the component.